routes/promotionRouter.js

```diff
@@ -0,0 +1,238 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const Promotion = require('../models/promotions');
+
+const promotionRouter = express.Router();
+
+promotionRouter.use(bodyParser.json());
+
+promotionRouter
```

Comment:
The formatting on these files differ. Next time, let's try to have a standard format for all files.